sortbootorder.c

```diff
@@ -76,6 +77,10 @@ int main(void) {
 	char *scon_str;
 	char *usb_str;
 
+	device_toggle[4] = 1;
+	device_toggle[5] = 1;
+	device_toggle[6] = 1;
```

Comment:
@macpijan let's use some defines instead of hardcoded array indexes. Add some description to indexes, where those came from.